index.html

```diff
@@ -7,70 +7,412 @@
+                                </li>
+                                <li>
+                                    <p class="tag bg-light-green">css</p>
+                                </li>
+                                <li>
+                                    <p class="tag bg-light-green">JavaScript</p>
+                                </li>
+                                <li>
+                                    <p class="tag bg-light-green">html</p>
+                                </li>
+                            </ul>
+                            <a class="button bg-green white card" href="#">See Project</a>
+                        </section>
+
+                    </section>
+                </section>
+                <section class="bordered-card">
+
+                    <section class="stretched-column card">
+                        <figure>
+
+                            <img src="../assets/place_holders/plaholder.png" class="bg-light-green place-holder" alt="">
+
+                        </figure>
+
+
+                        <section class="centered-column bg-white card">
+                            <header>
+                                <h3 class="minor-heading">
+                                    Multi-Post Stories <br> Gain+Glory
+                                </h3>
+                            </header>
+                            <ul class="tag-container">
+                                <li>
+                                    <p class="tag bg-light-green">Ruby on rails</p>
+                                </li>
+                                <li>
+                                    <p class="tag bg-light-green">css</p>
+                                </li>
+                                <li>
+                                    <p class="tag bg-light-green">JavaScript</p>
+                                </li>
+                                <li>
+                                    <p class="tag bg-light-green">html</p>
+                                </li>
+                            </ul>
+                            <a class="button bg-green white card" href="#">See Project</a>
+                        </section>
+
+                    </section>
+                </section>
+
+            </section>
+        </section>
+        <section class="about-me">
+            <section class="centered-column bg">
+
+                <header >
+                    <h2 class="dark-blue">
+                        About Me
+                    </h2>
+                    <section class="center">
+
+                        <h4 class="blue ">
+                            I can help you build a product , feature or website Look through some of my work and experience!
+                            If you like what you see and have a project you need coded,
+                            don’t hestiate to contact me.
+                        </h4>
```

Comment:
- [ ] Kindly wrap this text in a `p` tag instead of a `h4` tag for semantic reasons 👍